control-plane/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # go-discover builds the discover binary (which we don't currently publish
 # either).
 ARG GOLANG_VERSION
-FROM golang:${GOLANG_VERSION}-alpine as go-discover
+FROM golang:${GOLANG_VERSION}-alpine3.19 as go-discover
```

Comment:
No argument against pinning this, but I’m surprised we need to - it looks like `alpine` is already an alias for this tag. https://hub.docker.com/layers/library/golang/alpine/images/sha256-5a99b4049412cd34ad6b4e0c9527ae6beb9ae82d787b4bf3f4eff7aa13fc577a?context=explore Is a rebuild from the floating tag in itself actually the fix?